packages/uix-guest/src/guest.ts

```diff
@@ -248,6 +241,7 @@ export class Guest<
 
       this.hostConnectionPromise = hostConnectionPromise;
       this.hostConnection = await this.hostConnectionPromise;
+      this.emit("connected", { guest: this });
```

Comment:
We weren't emitting a local connected event in the guest! I discovered this when I tried to subscribe to it in order to set up the ResizeObserver for GuestUIFrame.